public/js/client.js

```diff
@@ -146,6 +146,26 @@ function addToCart(id) {
     });
 }
 
+// function update To Cart(itemId, quantity, callback)
+// cart/update request sent to frontend server (index.js - app.post("/cart/update" function...)
+function updateToCart(id, quantity, next) {
+
+	console.log("Sending request to update cart: item: " + id + " quantity: " + quantity);
```

Comment:
Probably best to remove this log message so we don't spam the user's console with "ok" messages.